### PR TITLE
fix: Fixed overwrite JSON type configuration being empty

### LINKED ISSUE #4483: 某些场景下覆盖json类型的配置会导致配置为空
- [x] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [x] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [x] 我已经仔细检查过[FAQ](https://www.apolloconfig.com/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

某些场景下覆盖json类型的配置会导致配置为空

**复现**

通过如下步骤可以复现：

1. 打开一个json类型的配置。
2. 使用ctrl+a快捷键全选并使用ctrl+c复制配置
3. 点击修改按钮
4. 在修改页面使用ctrl+a快捷键全选并使用ctrl+v粘贴。目的用于覆盖当前配置。
5. 点击提交，发现提交上去的是空数据，原来的数据丢失了。

**期望**
期望是覆盖而非数据丢失。

image

### 额外的细节和日志

- 版本：http://106.54.227.205/上可以复现。1.5.1上也可以复现。
- 错误日志
- 配置：
- 平台和操作系统

### PR DESCRIPTION
**Pull request by falser101**
## What's the purpose of this PR
bugfix

## Which issue(s) this PR fixes:
Fixes #4483

### CODE REVIEW

**Comment by nobodyiam** [comment]
This change seems to bring a new issue: if the user does want to clear the content, it won't work after this fix?

**Comment by falser101** [comment]
> if the user does want to clear the content

Yes, let me make a fix

**Comment by falser101** [comment]
> This change seems to bring a new issue: if the user does want to clear the content, it won't work after this fix?

Done.

File: apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js [hunk 1]
```diff
@@ -987,6 +987,21 @@ function directive($window, $translate, toastr, AppUtil, EventManager, Permissio
...
+                onChange: function (e) {
+                    if ((e[0].action === 'insert') && (scope.namespace.hasOwnProperty("editText")) && (scope.namespace.editText.length === 0)) {
+                        let text = ''
+                        for (let i = 0; i < e[0].lines.length; i++) {
+                            if (i === 0) {
+                                text = e[0].lines[0]
+                            } else {
+                                text += '\r\n' + e[0].lines[i]
+                            }
+                            console.log(text)
```
Comment (nobodyiam) [inline apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js:1000]: Is this console.log required?

Comment (falser101) [reply apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js]: Removed.

**Comment by nobodyiam** [comment]
Looks good to me, would you please also update the CHANGES.md?

**Comment by falser101** [comment]
> Looks good to me, would you please also update the CHANGES.md?

Done.

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**